slashing-protection/src/main/java/tech/pegasys/web3signer/slashingprotection/DbSlashingProtection.java

```diff
@@ -146,9 +147,13 @@ public void exportDataWithFilter(final OutputStream output, final List<String> p
 
   @Override
   public IncrementalExporter createIncrementalExporter(final OutputStream out) {
+    if (!gvrValidator.genesisValidatorRootExists()) {
+      return new NoOpIncrementalExporter(out);
```

Comment:
What does the output look like from this? I think we might be doing the wrong thing and returning an empty string. Teku I think is returning json with no data in this case https://github.com/ConsenSys/teku/blob/master/data/dataexchange/src/test/resources/emptySlashingData.json

Reply:
Updated to export metadata json without gvr and any data.